tests/rbac/common/addresser/task_owner_test.py

```diff
@@ -78,18 +78,16 @@ def test_address_random(self):
             addresser.address_is(rel_address2), addresser.AddressSpace.TASKS_OWNERS
         )
 
-    def test_address_static(self):
-        """Tests address makes the expected output given a specific input"""
-        task_id = "99968acb8f1a48b3a4bc21e2cd252e67"
-        user_id = "966ab67317234df489adb4bc1f517b88"
-        expected_address = (
-            "bac00100006666326a1713a905b26359fc8da23333cce7570f3f6f7d2c1635f6deea00"
-        )
-        rel_address = addresser.task.owner.address(
-            object_id=task_id, related_id=user_id
-        )
-        self.assertIsAddress(rel_address)
-        self.assertEqual(rel_address, expected_address)
-        self.assertEqual(
-            addresser.address_is(rel_address), addresser.AddressSpace.TASKS_OWNERS
-        )
+    def test_addresser_parse(self):
+        """Test addresser.parse returns a parsed address"""
+        task_id = addresser.task.unique_id()
```

Comment:
This is only compounding the problem we discussed earlier of having a static function nested deeply into an object hierarchy. We will need to extract it. Since I have already done the majority of work to that end, will have a PR soon.